package.json

```diff
@@ -14,17 +14,32 @@
   "main": "./bin/cli.js",
   "scripts": {
     "lint": "eslint ./lib",
-    "install-commit-validator": "fit-commit-js install"
+    "install-commit-validator": "fit-commit-js install",
+    "test": "jest"
```

Comment:
I'd also say that a watch for the test would be quite helpful when we want to code with TDD